packages/modules/web_themes/koala/source/src/stores/mqtt-store.ts

```diff
@@ -1615,6 +1616,88 @@ export const useMqttStore = defineStore('mqtt', () => {
     });
   };
 
+  /**
+   * Get the charge point connected vehicle bidi charging plan identified by the charge point id
+   * @param chargePointId charge point id
+   * @returns object | undefined
+   */
+  const chargePointConnectedVehicleBidiChargePlan = computed(() => {
+    return (chargePointId: number) => {
+      return chargePointConnectedVehicleChargeTemplate(chargePointId).value
+        ?.chargemode?.bidi_charging?.plan as BidiChargingPlan | undefined;
+    };
+  });
+
+  /**
+   * Get or set the charge point connected vehicle bidi charging minimaler Entlade-SoC identified by the charge point id
+   * @param chargePointId charge point id
+   * @returns number | undefined
+   */
+  const chargePointConnectedVehicleBidiChargeMinEntladeSoC = (
```

Comment:
Bitte kein "Denglisch", sondern nur englische Begriffe verwenden.
```suggestion
  const chargePointConnectedVehicleBidiChargeMinDischargeSoC = (
```